src/AspNet.Security.OpenIdConnect.Extensions/OpenIdConnectConstants.cs

```diff
@@ -22,22 +22,61 @@ public static class Algorithms {
         }
 
         public static class Claims {
-            public const string Active = "active";
-            public const string Address = "address";
-            public const string AtHash = "at_hash";
-            public const string ClientId = "client_id";
-            public const string Confidential = "confidential";
-            public const string EmailVerified = "email_verified";
-            public const string PhoneNumber = "phone_number";
-            public const string PhoneNumberVerified = "phone_number_verified";
-            public const string PreferredUsername = "preferred_username";
-            public const string Profile = "profile";
-            public const string Scope = "scope";
-            public const string TokenId = "token_id";
-            public const string TokenType = "token_type";
-            public const string Usage = "usage";
-            public const string Username = "username";
-            public const string Website = "website";
+            public static class Protocol {
+                public const string AtHash = "at_hash";
+                public const string Audience = "aud";
+                public const string AuthorizedParty = "azp";
+                public const string Confidential = "confidential";
+                public const string CryptographicHash = "c_hash";
```

Comment:
FYI, `c_hash` means `(authorization) code hash`, not `crypto hash` :smile: :smoking: